bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/ProcessJSON.java

```diff
@@ -0,0 +1,310 @@
+package org.dataflowanalysis.analysis.converter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.dataflowanalysis.analysis.converter.microsecend.*;
+import org.dataflowanalysis.analysis.converter.webdfd.*;
+import org.dataflowanalysis.dfd.datadictionary.*;
+import org.dataflowanalysis.dfd.dataflowdiagram.*;
+
+public class ProcessJSON {
+    private dataflowdiagramFactory dfdFactory;
+    private datadictionaryFactory ddFactory;
+
+    private Map<String, Node> nodesMap;
+    private Map<Node, List<String>> nodeToLabelNames;
+    private Map<String, LabelType> labelTypeMap;
```

Comment:
Do you use the values of `labelTypeMap` anywhere?

Reply:
Yes we do:
`labelTypeMap.put("annotation", annotation);`

Reply:
Yes, but the variable is never read from (e.g. returning to user, getting or removing elements)